packages/web-app/src/components/appli/Entry/Snapshots/index.jsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  AccordionActions,
+  Typography
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+
+import Alert from '../../../common/Alert';
+import Contribution from '../../../common/Contribution/Contribution';
+import { fetchSnapshot } from '../../../../actions/Snapshot/GetSnapshots';
+import GenericSnapshot from './GenericSnapshots';
+import REDUCER_STATUS from '../../../../reducers/ReducerStatus';
+import RiggingSnapshot from './RiggingSnapshots';
+import ScrollableContent from '../../../common/Layouts/Fixed/ScrollableContent';
+import getAuthor from '../../../../util/getAuthor';
+
+function renderComponent(type, data) {
+  switch (type) {
+    case 'rigging':
+      return <RiggingSnapshot rigging={data} key={data.id} />;
+    default:
+      // Todo: changer le default pour dire que ce n'est pas un truc pris en charge ou ajouter un warning sur la page générale
```

Comment:
comment in english ;)

Reply:
That's a todo i pushed by mistake, i will do it before merging that PR ! ^^